src/main/java/pages/login/ForgotPassword.java

```diff
@@ -14,29 +14,54 @@ public ForgotPassword () {
 	}
 	
 	public void initResentLink(){
-		emailAddressClick ();
-		sendResetLinkClick ();
-		backToLoginClick ();
+		setEmailAddress ();
+		setSendResetLink ();
+		setBackToLogin ();
 	}
 	
 	public WebElement emailAddressClick (){
-		emailAddress = WebDriver.driver.findElement ( By.cssSelector ( "#email" ) );
-		emailAddress.click ();
-		emailAddress.sendKeys ( "admin@gmail.com" );
+		getEmailAddress ().click ();
 		return emailAddress;
 	}
 	
-	public WebElement sendResetLinkClick (){
+	public ForgotLoginMessage sendResetLinkClick (){
+		getSendResetLink ().click ();
+		return new ForgotLoginMessage ();
+	}
+	
+	public WebElement backToLoginClick(){
+		getBackToLogin ().click ();
+		return backToLogin;
+	}
+	
+	public ForgotLoginMessage writeEmail(){
+		emailAddressClick ().sendKeys ( "admin@gmail.com" );
```

Comment:
maybe before typing email good to use method "clear" to be assure that field is empty

Reply:
Also
1. do not use gmail emails in tests. or any real domains at all. Only acceptable case are mail clients for testing, e.g. mainilator
2. do not hardcode emails inside methods. pass a parameter. your page object should not contain any input information